Composer/packages/server/src/controllers/project.ts

```diff
@@ -54,6 +55,33 @@ async function createProject(req: Request, res: Response) {
     await AssetService.manager.copyBoilerplate(currentProject.dataDir, currentProject.fileStorage);
 
     if (currentProject !== undefined) {
+      if (currentProject?.settings?.runtime.customRuntime === true) {
```

Comment:
you just made sure currentProject is defined, .? is unnecessary

Reply:
can a project not have settings?

Reply:
Removed the null checks, first is not needed and all bot projects are loaded with default settings if their template does not prepopulate